src/server/set_family.cc

```diff
@@ -1435,18 +1435,28 @@ void SScan(CmdArgList args, const CommandContext& cmd_cntx) {
 
 // Syntax: saddex key ttl_sec member [member...]
 void SAddEx(CmdArgList args, const CommandContext& cmd_cntx) {
-  string_view key = ArgS(args, 0);
-  string_view ttl_str = ArgS(args, 1);
-  uint32_t ttl_sec;
-  constexpr uint32_t kMaxTtl = (1UL << 26);
+  CmdArgParser parser(args);
+
+  auto key = parser.Next<std::string_view>();
```

Comment:
remember to use explicit types for simple types

Reply:
fixed